src/main/java/com/microsoft/graph/concurrency/ChunkedUploadResponseHandler.java

```diff
@@ -22,26 +22,35 @@
 
 package com.microsoft.graph.concurrency;
 
-import java.io.BufferedInputStream;
-import java.io.InputStream;
-import java.util.Map;
+import com.google.common.io.ByteStreams;
 
-import javax.annotation.Nullable;
-import javax.annotation.Nonnull;
-
-import com.microsoft.graph.http.CoreHttpProvider;
+import com.microsoft.graph.core.ClientException;
 import com.microsoft.graph.core.Constants;
+import com.microsoft.graph.http.CoreHttpProvider;
 import com.microsoft.graph.http.GraphServiceException;
-import com.microsoft.graph.core.ClientException;
 import com.microsoft.graph.http.HttpResponseCode;
 import com.microsoft.graph.http.HttpResponseHeadersHelper;
 import com.microsoft.graph.http.IHttpRequest;
 import com.microsoft.graph.http.IStatefulResponseHandler;
 import com.microsoft.graph.logger.ILogger;
 import com.microsoft.graph.serializer.ISerializer;
 
+import org.jetbrains.annotations.NotNull;
```

Comment:
Isn't `org.jetbrains.annotations.NotNull` an IDE specific annotation? Can we use javax.annotation.Nonnull for this?

Reply:
https://github.com/microsoftgraph/msgraph-sdk-java-core/pull/123/files#diff-730fef586e049fb41d489bf9c76db82d56e0f1c5ee1732d0730adf28d9f53d94R131

Reply:
Oops, IntelliJ added some automatically as I was refactoring code. Thanks for pointing this out!